spring-data-relational/src/main/java/org/springframework/data/relational/repository/query/RelationalExampleMapper.java

```diff
@@ -22,23 +22,31 @@
 import java.util.List;
 import java.util.Optional;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
We don't want to have Jetbrains annotations here.

Reply:
Sure, my bad, removed